views/src/main/java/com/lush/lib/handler/LinkHandler.kt

```diff
@@ -0,0 +1,70 @@
+package com.lush.lib.handler
+
+import android.content.ActivityNotFoundException
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+
+/**
+ * Handler class to handle telephone, navigation and e-mail links
+ *
+ * @author Gokhan
+ */
+class LinkHandler
+{
+	companion object
+	{
+		@JvmStatic
```

Comment:
Please remove JVMStatic and companion object uses

Reply:
Once you remove both, you can access methods by calling `LinkHandler.INSTANCE.handleLink()`